### DISCUSSION TITLE
I got a little problem when reinstalling filles

### DISCUSSION THREAD

**Discussion by superAptosaur** [Monika-After-Story/MonikaModDev#9489] [QA - tech support + general] [answered]
Hey,

so in the past i tried to install a mod where i could change the background to look more like a living room, but i installed it incorrectly and a lot of files disappeared, as i always do when i can't solve some problems, i wait a bit until i can solve it later. while waiting and searching on the internet, i got the idea to reinstall everything again, so i did it. but now i'm missing only one file and it's the english translation file. i tried so many things like installing ddlc again, but nothing changes. Should I try again?

it looks like this:

'' I'm sorry, but an uncaught exception occurred.

While running game code:
Exception: Could not load from archive tl/english/script.rpyc.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\superAptosaur\Downloads\goofy aaah\DDLC-1.1.1-pc\renpy\bootstrap.py", line 295, in bootstrap
    renpy.main.main()
  File "C:\Users\superAptosaur\Downloads\goofy aaah\DDLC-1.1.1-pc\renpy\main.py", line 364, in main
    renpy.game.script.load_script()  # sets renpy.game.script.
  File "C:\Users\superAptosaur\Downloads\goofy aaah\DDLC-1.1.1-pc\renpy\script.py", line 265, in load_script
    self.load_appropriate_file(".rpyc", ".rpy", dir, fn, initcode)
  File "C:\Users\superAptosaur\Downloads\goofy aaah\DDLC-1.1.1-pc\renpy\script.py", line 667, in load_appropriate_file
    raise Exception("Could not load from archive %s." % (lastfn,))
Exception: Could not load from archive tl/english/script.rpyc.

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.9 ''

dont question the map name.

**Comment by Wingdinggaster656** [comment]
So you are using a translation version? If that's the case, I'm afraid we can't offer perfect help.

Possible solution: Reinstall your translation patches as well.
You may also need to ask the translators.

**Comment by superAptosaur** [comment]
no, i did not install any translations
(sorry for the late response)

**Comment by superAptosaur** [comment]
isnt there a way to install that missing script and put it directly in the game folders?

**Comment by Wingdinggaster656** [comment]
It's really weird. We probably don't have a English translation file. Are you sure you did a 100% reinstall?

**Comment by superAptosaur** [comment]
Yes, i tried it 2 times. The first time it just opened the normal ddlc even with the MAS filles :/
so I tried again and now it just wont open.

**Comment by Wingdinggaster656** [comment]
I think the first time is actually almost a success. You should try to reinstall the game 3rd time.

**Comment by Booplicate** [comment, marked as answer]
That's definitely not our official build.
- download MAS from [the official site](http://monikaafterstory.com/releases.html)
- download DDLC from [the official site](https://ddlc.moe/)
- make a new clean install
- your progress may or may not transfer since you've used an unofficial build

**Reply by superAptosaur** [reply]
I tried 3 times and still no succes, It opens normal ddlc again. weirdly it didnt even ask me to remplace filles when i put the newest MAS filles in ddlc. Do i do something wrong?

**Reply by Booplicate** [reply]
Right now MAS should be installed into the `game/` folder. If you get DDLC, then you extracted it somewhere else.

**Reply by superAptosaur** [reply]
YESSSS, it works! and its still stayed the same with the name to the outfit i gave her. i dont know how to thank you 😭. (I didnt think it was so simple to fix)
